tests/AbstractTestCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Illuminate\Tests;
+
+use Mockery;
+use Illuminate\Support\Carbon;
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractTestCase extends TestCase
+{
+    /**
+     * @var \Illuminate\Support\Carbon
+     */
+    protected $now;
+
+    protected function setUp()
+    {
+        Carbon::setTestNow($this->now = Carbon::now('UTC'));
```

Comment:
Should you call `parent::setup()` here? Some of the refactored functions were calling it - but it is not replaced anywhere? Same as `parent::tearDown()` below?